core/templates/domain/collection/collection-rights-backend-api.service.spec.ts

```diff
@@ -41,11 +41,11 @@ describe('Collection rights backend API service', function() {
       TestBed.get(CollectionRightsBackendApiService);
     csrfService = TestBed.get(CsrfTokenService);
 
-    spyOn(csrfService, 'getTokenAsync').and.callFake(() => {
-      return new Promise((resolve) => {
-        resolve('sample-csrf-token');
-      });
-    });
+    // spyOn(csrfService, 'getTokenAsync').and.callFake(() => {
+    //   return new Promise((resolve) => {
+    //     resolve('sample-csrf-token');
+    //   });
+    // });
```

Comment:
Why not remove this? Or if we except it to be enabled again the add TODO.

Reply:
Done!